### PR TITLE
[15.0] Add some departments fr oversea

### PR DESCRIPTION
**Pull request by DilawBz**
We have added in this commit some departments of france oversea which were not included in your versions

### CODE REVIEW

File: l10n_fr_department_oversea/data/res_country_department.xml [hunk 1]
```diff
@@ -24,9 +24,57 @@
         <field name="code">974</field>
         <field name="state_id" ref="res_country_state_reunion" />
     </record>
+    <!-- ### ADD AUNEOR-CONSEIL ### -->
```
Comment (legalsylvain) [inline l10n_fr_department_oversea/data/res_country_department.xml]: Please remove the lines regarding auneor.

File: l10n_fr_department_oversea/data/res_country_department.xml [hunk 2]
```diff
@@ -24,9 +24,57 @@
...
+        <field name="name">Polynésie Française</field>
+        <field name="code">987</field>
+        <field name="state_id" ref="res_country_state_polynesie_francaise" />
+    </record>
+    <record id="res_country_nouvelle_caledonie" model="res.country.department">
+        <field name="name">Nouvelle Calédonie</field>
+        <field name="code">988</field>
+        <field name="state_id" ref="res_country_state_nouvelle_caledonie" />
+    </record>
+    <record id="res_country_department_monaco" model="res.country.department">
```
Comment (legalsylvain) [inline l10n_fr_department_oversea/data/res_country_department.xml]: Monaco is a country.

**Review by legalsylvain — CHANGES_REQUESTED** [review]
Thanks for jour contribution. Some blocking points inline.
Regards.

**Comment by alexis-via** [comment]
I disagree with this PR ; so far, res.country.department was for real French departments (DOMs) and not TOMs. I think we should not change that.

**Comment by alexis-via** [comment]
So I propose to close this PR.

**Pull request closed without merging**